platform/10/_index.md

```diff
@@ -1,28 +1,29 @@
 ---
-title: "Jakarta EE Platform 10 (under development)"
-date: 2021-08-11
-summary: "Jakarta EE Platform 10 defines a standard platform for hosting Jakarta EE applications. Find specifications and compatible implementations."
-seo_title: "Jakarta EE Platform 10 (Under Development) | Standards for hosting Jakarta EE applications"
+title: "Jakarta EE Platform 10"
+date: 2022-06-09
+summary: "Release of the Jakarta EE 10 Platform"
 ---
 The Jakarta EE Platform defines a standard platform for hosting Jakarta EE applications.
 
+# Artifacts
+
 * [Jakarta EE Platform 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/10)
   * [Staged Jakarta EE 10 Release Plan](https://docs.google.com/document/d/1U24VmTzAfXcn3WBnVcolb8vhZO-Pnk_bit0CKh_d2jM/edit#)
   * [Final Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
-* [Jakarta EE Platform 10 Specification Document](./jakarta-platform-spec-10.pdf) (PDF)
-* [Jakarta EE Platform 10 Specification Document](./jakarta-platform-spec-10.html) (HTML)
+* [Jakarta EE Platform 10 Specification Document](./jakarta-platform-spec-10.0.pdf) (PDF)
+* [Jakarta EE Platform 10 Specification Document](./jakarta-platform-spec-10.0.html) (HTML)
 * [Jakarta EE Platform 10 Javadoc](./apidocs)
 * [Jakarta EE Platform 10 TCK](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sha256),[pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
 * Maven coordinates
   * [jakarta.platform:jakarta.jakartaee-api:jar:10.0.0](https://search.maven.org/artifact/jakarta.platform/jakarta.jakartaee-api/10.0.0/jar)
 * Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
-  * [TBD]()
+  * [GlassFish-7.0.0-M7](https://download.eclipse.org/ee4j/glassfish/glassfish-7.0.0-M7.zip)
```

Comment:
Update to M8

Reply:
Link to M8: https://download.eclipse.org/ee4j/glassfish/glassfish-7.0.0-M8.zip

Reply:
This should be ready